<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewUser</class>
 <widget class="QDialog" name="AddNewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/icons/Logo2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>551</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New User</string>
   </property>
   <widget class="QLabel" name="lbl_first_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Name*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Email*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_phone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Password*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Username*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_phone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Phone*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_avatar">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Avtar*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_okay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/icons/interfaces.png</normaloff>:/resources/icons/interfaces.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/icons/signs (3).png</normaloff>:/resources/icons/signs (3).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_confirm_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Confirm Password*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>User Role*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_user_role">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select user role</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cashier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Supervisor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>superadmin</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_image">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonAvatar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open...</string>
    </property>
   </widget>
   <widget class="QLabel" name="avatar_display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Set User Status*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_user_status">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select User Status</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>active</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>under evaluation</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/background/picklit.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
